zookeeper-metrics-providers/zookeeper-prometheus-metrics/src/main/java/org/apache/zookeeper/metrics/prometheus/PrometheusMetricsProvider.java

```diff
@@ -91,6 +91,11 @@ public class PrometheusMetricsProvider implements MetricsProvider {
      */
     static final String WORKER_SHUTDOWN_TIMEOUT_MS = "workerShutdownTimeoutMs";
 
+    /**
+     * The interval in seconds for Prometheus summary metrics rotation. Default value is 60.
+     */
+    static final String PROMETHEUS_SUMMARY_ROTATE_SECONDS = "prometheusMetricsSummaryRotateSeconds";
```

Comment:
Can you update Zookeeper admin doc to include the property? https://zookeeper.apache.org/doc/r3.9.3/zookeeperAdmin.html#Debug+Log+Cleanup+Logback

Reply:
updated